mitmproxy/proxy/commands.py

```diff
@@ -87,45 +86,16 @@ def __init__(self, connection: Connection, half_close: bool = False):
         self.half_close = half_close
 
 
-class Hook(Command):
+class Hook(Command, mitmproxy.events.MitmproxyEvent):
```

Comment:
nitpick: I think everywhere else we use sub-import, `from mitmproxy import events`, instead of the full path...

Reply:
Yes. I made it explicit here because the _proxy core events_ have nothing to do with the _mitmproxy events_, which unfortuately is really ugly. (see my comment at https://github.com/mitmproxy/mitmproxy/pull/4377#discussion_r550805197)